chingsley/ireporter
# Ch records controller

#### What does this PR do?
This pull request deals with code maintainability

#### Description of Task to be completed?
- set up outputFormatter function in the middleware to handle repeated output formatting

#### How should this be manually tested?
- clone the repo
- cd into the root directory
- run `npm install' to install the necessary depencies
- run `npm test' to see the tests pass
- run `npm start' to start the server
- using Postman, check out the routes listed in README.md
